monai/transforms/intensity/array.py

```diff
@@ -2161,3 +2157,97 @@ def __call__(self, img: torch.Tensor) -> torch.Tensor:
                 img = IntensityRemap(self.kernel_size, self.R.choice([-self.slope, self.slope]))(img)
 
         return img
+
+
+class ForegroundMask(Transform):
+    """
+    Creates a binary mask that defines the foreground based on thresholds in RGB or HSV color space.
+    This transform receives an RGB (or grayscale) image where by default it is assumed that the foreground has
+    low values (dark) while the background has high values (white). Otherwise, set `invert` argument to `True`.
+
+    Args:
+        threshold: an int or a float number that defines the threshold that values less than that are foreground.
+            It also can be a callable that receives each dimension of the image and calculate the threshold,
+            or a string the defines such callable from `skimage.filter.threshold_xxxx`.
+            Moreover, a dictionary can be passed that defines such thresholds for each channel, like
+            {"R": 100, "G": "otsu", "B": skimage.filter.threshold_mean}
```

Comment:
Could you please help add the reference link of `skimage.filter.threshold_XXX` here?

Thanks.